Packs/PiHole/Integrations/PiHole/PiHole.py

```diff
@@ -0,0 +1,151 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+''' IMPORTS '''
+
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' CONSTANTS '''
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+
+class Client:
+    def __init__(self, base_url, verify, proxies, auth):
+        self.base_url = base_url
+        self.verify = verify
+        self.proxies = proxies
+        self.auth = '&auth=' + auth
+
+    def http_request(self, method, url_suffix):
+        server = self.base_url + url_suffix + self.auth
+        res = requests.request(
+            method,
+            server,
+            verify=self.verify,
+            proxies=self.proxies
+        )
+        try:
+            return res.json()
+        except ValueError:
+            data = {'Info': res.text}
+            return data
+
+
+def results_return(command, thingtoreturn):
+    finaldata = {'PiHole': {command: thingtoreturn}}
+    return demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': thingtoreturn,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown(command, thingtoreturn, removeNull=True),
+        'EntryContext': finaldata
+    })
+
+
+def test_module(client):
+    """
+    Returning 'ok' indicates that the integration works like it is supposed to. Connection to the service is successful.
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+    result = client.http_request('GET', '')
```

Comment:
what does that request do ?

Reply:
It just get the base url of the api to see it responds correctly.